src/core/reactor_backend.cc

```diff
@@ -228,21 +228,21 @@ aio_storage_context::submit_work() {
 }
 
 void aio_storage_context::schedule_retry() {
-    // loop until both _pending_aio_retry and _aio_retries are empty.
-    // While retrying _aio_retries, new retries may be queued onto _pending_aio_retry.
+    // loop until both _pending_aio_retry and _aio_rentries are empty.
```

Comment:
This fix is wrong. "_aio_retries" means retries (plural of retry) of aio (asynchronous IO). It's not "rentries", whatever that is supposed to be.

Reply:
Hmm, indeed. This is damage from 9cefe7f, I'll make sure to tease that out.